test/boulder-config.json

```diff
@@ -172,5 +172,5 @@
     "dnsTimeout": "10s"
   },
 
-  "subscriberAgreementURL": "http://localhost:4300/terms"
+  "subscriberAgreementURL": "http://localhost:4000/directory"
```

Comment:
I came up with a simple fix but since it will conflict with this branch it's probably easier for you just to integrate it here. Add to startservers.py:

```
import BaseHTTPServer

class ToSServer(BaseHTTPServer.BaseHTTPRequestHandler):
    def do_GET(self):
        self.send_response(200)
        self.end_headers()
        self.wfile.write("Do What Ye Will (An it Harm None).\n")
httpd = SocketServer.TCPServer(("localhost", 4001), ToSServer)
httpd.serve_forever()
```
Then edit boulder-config.json to point subscriberAgreementURL to 'http://localhost:4001/terms/v1' (the provided HTTP server will serve up terms for any path, but this reflects a similar versioning style to what we might use in prod).